examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -101,33 +89,46 @@ def __init__(self, before, after, gradient, update):
 
     # TODO: def __str__
 
-    def save(self, name='parameter_update.npz'):
-        """save numpy arrays to path/to/results/{name}.npz"""
-        parameter_update_path = path_to_results.joinpath(name)
-        print(f'saving parameter updates to {parameter_update_path}')
+    def save(self, fname='parameter_update.npz'):
+        """save numpy arrays to fname"""
+        print(f'saving parameter updates to {fname}')
         np.savez(
-            file=parameter_update_path,
+            file=fname,
             before=self.before,
             after=self.after,
             gradient=self.gradient,
             update=self.update,
         )
 
 
-def _save_forcefield(filename, ff_params):
-    # TODO: update path
-
-    with open(path_to_results.joinpath(filename), 'w') as fh:
+def _save_forcefield(fname, ff_params):
+    with open(fname, 'w') as fh:
         fh.write(ff_params)
 
 
 if __name__ == "__main__":
+    import datetime
```

Comment:
nit: group imports together at top?

Reply:
+1

Only time imports should be conditional is if there's dependency issues

Reply:
Resolved in https://github.com/proteneer/timemachine/pull/367/commits/a24361543e280abcd444aa08f756fb96489d3557